Samples/Swift/DaysUntilBirthday/Podfile.lock

```diff
@@ -0,0 +1,61 @@
+PODS:
```

Comment:
We'll probably want to exclude this file.

Reply:
Given that were are grabbing the dependency via a local path, I think it makes sense to remove the `Podfile.lock`.